composer.json

```diff
@@ -5,7 +5,8 @@
     "license":     "BSD-3-Clause",
     "type":        "project",
     "autoload": {
-        "psr-0": { "": "src/", "BecklynSymfony": "app/" }
+        "psr-4": { "": "src/", "BecklynSymfony\\": "app/BecklynSymfony/" },
+        "files": [ "app/AppKernel.php" ]
```

Comment:
This is only needed if we update everything to the new standard edition (which is still on the roadmap).